hermes-client/src/main/java/pl/allegro/tech/hermes/client/metrics/MicrometerMetricsProvider.java

```diff
@@ -17,30 +18,40 @@ public MicrometerMetricsProvider(MeterRegistry metrics) {
     }
 
     @Override
-    public void counterIncrement(String name) {
-        metrics.counter(name).increment();
+    public void counterIncrement(String topic, String key) {
+        counterIncrement(topic, key, new HashMap<>());
```

Comment:
I suggest replacing `new HashMap<>()` with `Collections.emptyMap()` - just to be a little bit gc friendly :)

Reply:
Take a look at the called function. I am calling `put` at this object. As far as I know, `put` called on `Collection.emptyMap()` will cause an error.